src/app-layout/__tests__/common.test.tsx

```diff
@@ -253,7 +254,7 @@ describeEachAppLayout(size => {
           items: singleDrawer.drawers.items,
         },
       };
-      const { wrapper } = renderComponent(<AppLayout contentType="form" {...drawersClosed} />);
+      const { wrapper } = renderComponent(<AppLayout contentType="form" {...(drawersClosed as any)} />);
```

Comment:
Because drawers API is now public, there is a type error in all existing tests using beta API.

I updated all tests where it was an easy 1-line migration. For more complex cases, let's do it as a separate follow up PR and use `as any` for now